app/models/address.rb

```diff
@@ -1,15 +1,23 @@
+require 'geocoder'
+
 class Address < ActiveRecord::Base
-  def self.find_or_create_from_params(params)
-    find_or_create_by!(
-      address_line_1: params['address_line_1'].presence,
-      address_line_2: params['address_line_2'].presence,
-      address_line_3: params['address_line_3'].presence,
-      town: params['town'].presence,
-      county: params['county'].presence,
-      postcode: params['postcode'].presence
-    )
+  class << self
```

Comment:
Cool. We should cover this with a rubocop rule at some point in the not too distant future.